segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/DebugStreamSegmentContainerTests.java

```diff
@@ -324,7 +324,7 @@ public void testDataRecoveryContainerLevel() throws Exception {
 
         @Cleanup
         TestContext context = createContext(executorService());
-        OperationLogFactory localDurableLogFactory = new DurableLogFactory(DEFAULT_DURABLE_LOG_CONFIG, context.dataLogFactory, executorService());
```

Comment:
Why this is solving the problem? Please, describe this in detail in the PR, as we need to understand the purpose of the change. Also, let us know how have you verified that the original problem has been solved, as the fact that the build passes (how many times? in which environments?) does not mean that the unreliability of the test has been solved.

Reply:
Hi Raul

Yes sure. It was the only problem I could find in the test then. Still working on it.